### ISSUE TITLE
v2.0.0 - Cannot set property bytesRead of #<Socket> which has only a getter

### ISSUE THREAD

**Issue by Mithrilhall** [gpoitch/ember-cli-less#65]
```
$ ember -v
ember-cli: 3.2.0
node: 8.11.3
os: linux x64
```

This only happened to me when using version v2.0.0. Once I rolled back to v1.5.7 the error was gone.

Steps to reproduce error:

```
ember new ember-cli-less-test
cd ember-cli-less-test
ember install ember-cli-less
mv app/styles/app.css app/styles/app.less
ember s
```

Error:
```
Build Error (LessCompiler)

Cannot set property bytesRead of #<Socket> which has only a getter

Stack Trace and Error Report: /tmp/error.dump.abbfe53ad932dff8b89d3c69328a7696.log
```

**Comment by knownasilya** [comment]
Could you past the contents of `/tmp/error.dump.abbfe53ad932dff8b89d3c69328a7696.log`?

**Comment by knownasilya** [comment]
cc @jasonmit

**Comment by Mithrilhall** [comment]
```
=================================================================================

ENV Summary:

  TIME: Fri Aug 10 2018 10:36:02 GMT-0400 (EDT)
  TITLE: ember
  ARGV:
  - /home/robert/.nvm/versions/node/v8.11.3/bin/node
  - /home/robert/.nvm/versions/node/v8.11.3/bin/ember
  - s
  EXEC_PATH: /home/robert/.nvm/versions/node/v8.11.3/bin/node
  TMPDIR: /tmp
  SHELL: /bin/bash
  PATH:
  - /home/robert/.nvm/versions/node/v8.11.3/bin
  - /home/robert/bin
  - /home/robert/.local/bin
  - /usr/local/sbin
  - /usr/local/bin
  - /usr/sbin
  - /usr/bin
  - /sbin
  - /bin
  - /usr/games
  - /usr/local/games
  - /snap/bin
  PLATFORM: linux x64
  FREEMEM: 2627375104
  TOTALMEM: 16209571840
  UPTIME: 853380
  LOADAVG: 1.77490234375,1.763671875,2.0791015625
  CPUS:
  - Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz - 2656
  - Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz - 2635
  - Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz - 2683
  - Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz - 2695
  ENDIANNESS: LE
  VERSIONS:
  - ares: 1.10.1-DEV
  - cldr: 32.0
  - http_parser: 2.8.0
  - icu: 60.1
  - modules: 57
  - napi: 3
  - nghttp2: 1.32.0
  - node: 8.11.3
  - openssl: 1.0.2o
  - tz: 2017c
  - unicode: 10.0
  - uv: 1.19.1
  - v8: 6.2.414.54
  - zlib: 1.2.11

ERROR Summary:

  - broccoliBuilderErrorStack: TypeError: Cannot set property bytesRead of #<Socket> which has only a getter
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:16)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
  - codeFrame: Cannot set property bytesRead of #<Socket> which has only a getter
  - errorMessage: Build Canceled: Broccoli Builder ran into an error with `LessCompiler` plugin. 💥
Cannot set property bytesRead of #<Socket> which has only a getter
  - errorType: Build Error
  - location:
    - column: [undefined]
    - file: [undefined]
    - line: [undefined]
    - treeDir: [undefined]
  - message: Build Canceled: Broccoli Builder ran into an error with `LessCompiler` plugin. 💥
Cannot set property bytesRead of #<Socket> which has only a getter
  - name: Error
  - nodeAnnotation: LessCompiler
  - nodeName: LessCompiler
  - originalErrorMessage: Cannot set property bytesRead of #<Socket> which has only a getter
  - stack: TypeError: Cannot set property bytesRead of #<Socket> which has only a getter
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:16)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)
    at _clone (/home/robert/source-local/ember-cli-less-test/node_modules/clone/clone.js:162:18)

=================================================================================

```

**Comment by knownasilya** [comment]
Looks like it's blowing up in the `clone` dep, which is used by `less` directly.

**Comment by jasonmit** [comment]
https://github.com/gpoitch/ember-cli-less/pull/66 resolves this

**Comment by knownasilya** [comment]
Published as v2.0.1
